contents/docs/libraries/ios/index.mdx

```diff
@@ -13,7 +13,7 @@ features:
   sessionRecording: true
   featureFlags: true
   groupAnalytics: true
-  surveys: false
+  surveys: true
```

Comment:
on this page you can also add a surveys section see here https://posthog.com/docs/libraries/react-native#surveys